docker-compose.identity.yml

```diff
@@ -0,0 +1,24 @@
+# extends docker-compose.yml
+# https://docs.docker.com/compose/extends/#multiple-compose-files
+
+version: '2'
+
+services:
+  identity:
+    image: wso2is:5.5.0
```

Comment:
Note that this version doesn't match what's in use elsewhere, but seems to be ok...?

Reply:
Noted in the docs.